packages/workshop-app/tailwind.config.js

```diff
@@ -20,4 +25,9 @@ module.exports = {
 		require('tailwindcss-radix'),
 		require('tailwind-scrollbar'),
 	],
+	safelist: [
+		{
+			pattern: /(bg|text|border)-(red|green|blue)-500/,
+		},
+	],
```

Comment:
Why do we need this? I'd rather not have this.

Reply:
I use dynamic classes in `toast.tsx`

`variantColor ? `border-l-8 border-${variantColor}-500` : '',`

`variantColor ? `text-${variantColor}-500` : ''`

if we fox example don't have the classes border-red-500 it the code then tailwind post css don't include it in the generated css file. 
If use prefer i cam use clsx for each color